### PR TITLE
Support Istio 1.8.0

### PR DESCRIPTION
**Pull request by waynz0r**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | yes
| Related tickets | 
| License         | Apache 2.0

### What's in this PR?

- Support for Istio `1.8.x` and make `1.8.0` the default version on the `release-1.8` branch

### CODE REVIEW

File: docs/upgrade/canary-upgrade.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 ## High level flow
 
-Let us suppose that we have an Istio 1.6 control plane running in our cluster and we would like to upgrade to an Istio 1.7 control plane.
+Let us suppose that we have an Istio 1.6 control plane running in our cluster and we would like to upgrade to an Istio 1.8 control plane.
```
Comment (khernyo) [inline docs/upgrade/canary-upgrade.md:5]:
```suggestion
Let us suppose that we have an Istio 1.7 control plane running in our cluster and we would like to upgrade to an Istio 1.8 control plane.
```
And similarly for the rest of this doc.

Comment (khernyo) [reply docs/upgrade/canary-upgrade.md]: There are other such cases in both canary-upgrade.md and in-place-upgrade.md

Comment (waynz0r) [reply docs/upgrade/canary-upgrade.md]: 👍

**Review by khernyo — APPROVED** [review]

**Review by Laci21 — APPROVED** [review]
LGTM, thank you @waynz0r!

Only a few nits as discussed offline:
- Use ghcr images
- Update Istio CRDs
- Update Helm charts

**Review by Laci21 — APPROVED** [review]

**Pull request merged**